cdc/sorter/memory/entry_sorter.go

```diff
@@ -159,25 +130,57 @@ func (es *EntrySorter) Run(ctx context.Context) error {
 }
 
 // AddEntry adds an RawKVEntry to the EntryGroup
-func (es *EntrySorter) AddEntry(ctx context.Context, entry *model.PolymorphicEvent) {
+func (es *EntrySorter) AddEntry(_ context.Context, entry *model.PolymorphicEvent) {
 	if atomic.LoadInt32(&es.closed) != 0 {
 		return
 	}
 	es.lock.Lock()
+	defer es.lock.Unlock()
 	if entry.RawKV.OpType == model.OpTypeResolved {
 		es.resolvedTsGroup = append(es.resolvedTsGroup, entry.CRTs)
 		es.resolvedNotifier.Notify()
 	} else {
 		es.unsorted = append(es.unsorted, entry)
 	}
-	es.lock.Unlock()
 }
 
 // Output returns the sorted raw kv output channel
 func (es *EntrySorter) Output() <-chan *model.PolymorphicEvent {
 	return es.outputCh
 }
 
+func eventLess(i *model.PolymorphicEvent, j *model.PolymorphicEvent) bool {
```

Comment:
Is it better that use e1, e2 as args name instead of i, j?

Reply:
i,j shows the index order, which is a bit more intuitive.